proto/token.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+import "permissions.proto";
+
+option go_package = "github.com/momentohq/client-sdk-go;client_sdk_go";
+option java_multiple_files = true;
+option java_package = "momento.auth";
+
+package token;
```

Comment:
Same for this - do we get any real benefit from using packages for new proto files? grpc-web + strictjs doesn't like them. Also nit: the java package doesn't match.